src/Organic/AdminSettings.php

```diff
@@ -77,11 +77,6 @@ protected function updateSettings() {
         }
 
         // Organic Settings
-        $this->organic->updateOption(
-            'organic::enabled',
-            isset( $_POST['organic_enabled'] ) ? true : false,
-            false
-        );
```

Comment:
"Enabled" will be offloaded to WordPress, i.e., plugin is activated or deactivated. If activated, we will now have more granular options about what, exactly, is enabled, e.g., Ads, Affiliate, etc.

Reply:
Nevermind, see https://github.com/orgnc/wordpress-plugin/pull/183#discussion_r1254378768